### PR TITLE
update TOC and change title of setup mac/linux article

### PR DESCRIPTION
**Pull request by sdwheeler**
Adding new PSCore v6 articles to TOC and fixing one title

Version(s) of document impacted
------------------------------
- [X] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [X] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/docs-conceptual/toc.yml [hunk 1]
```diff
@@ -72,8 +72,17 @@
         href: core-powershell/ise/windows-powershell-ise-object-model-reference.md
     - name: ISE Guide
       href: core-powershell/ise-guide.md
+    - name: Visual Studio Code
+      items:
+      - name: Using Visual Studio Cod
```
Comment (joeyaiello) [inline reference/docs-conceptual/toc.yml]: Should be `Code`

File: reference/docs-conceptual/toc.yml [hunk 2]
```diff
@@ -72,8 +72,17 @@
...
+        href: core-powershell/vscode/using-vscode.m
```
Comment (joeyaiello) [inline reference/docs-conceptual/toc.yml]: Should be `.md`

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-v6fixes/reference/docs-conceptual/toc.yml) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Ftoc.yml)
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-v6fixes/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md) | :white_check_mark:Succeeded | |

<a name="reference%2Fdocs-conceptual%2Ftoc.yml"></a>
### [reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-v6fixes/reference/docs-conceptual/toc.yml)
 - **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/vscode/using-vscode.m).
 - **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/vscode/using-vscode.m).
 - **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/vscode/using-vscode.m).
 - **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/vscode/using-vscode.m).
 - **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/vscode/using-vscode.m).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801102144356665-2030%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=xUElmL%2FVUE4Om7EqzJLH5br74NgeTHBMkZ40BEKlevY%3D&st=2018-01-10T22%3A07%3A22Z&se=2018-02-10T22%3A12%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-v6fixes/reference/docs-conceptual/setup/Installing-PowerShell-Core-on-macOS-and-Linux.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/toc.yml](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-v6fixes/reference/docs-conceptual/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801102157418567-2030%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=XLAL7HQf0Utc79tVDI6RHmkjQl6YgRUo2eD6ZbFKiWQ%3D&st=2018-01-10T22%3A34%3A47Z&se=2018-02-10T22%3A39%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**